packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -532,6 +535,13 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
         'existing handle will leak into another test and alter its behavior.'
       );
     }
+    _semanticsHandles = null;
+  }
+
+  int _semanticsHandles;
```

Comment:
I'd use a more specific name. As is it implies the current number of semantic handles, while you want to convey something like "last recorded", or "before test started".